### PR TITLE
Added 2020 Nissan Altima fingerprint

### PR DESCRIPTION
**Pull request by avolmensky**
Fingerprint for 2020 Nissan Altima fingerprint (Platimum 1.8 turbo)

### CODE REVIEW

File: selfdrive/car/nissan/values.py [hunk 1]
```diff
@@ -52,6 +52,10 @@ class CAR:
     {
       2: 5, 42: 6, 346: 6, 347: 5, 348: 8, 349: 7, 386: 8, 397: 8, 398: 8, 520: 2, 523: 6, 548: 8, 634: 7, 645: 8, 658: 8, 665: 8, 666: 8, 674: 2, 682: 8, 683: 8, 689: 8, 723: 8, 758: 3, 772: 8, 773: 6, 774: 7, 775: 8, 776: 6, 777: 7, 778: 6, 783: 3, 851: 8, 855: 5, 1001: 6, 1041: 8, 1042: 8, 1055: 3, 1100: 7, 1104: 4, 1105: 6, 1107: 4, 1108: 8, 1110: 7, 1111: 7, 1227: 8, 1228: 8, 1229: 8, 1232: 8, 1247: 4, 1266: 8, 1273: 7, 1306: 1, 1342: 1, 1376: 8, 1401: 8, 1497: 3, 1514: 6, 1526: 8, 1527: 5, 1792: 8, 1821: 8, 1823: 8, 1837: 8, 1872: 8, 1937: 8, 1953: 8, 1968: 8, 1988: 8, 2000: 8, 2001: 8, 2004: 8, 2005: 8, 2015: 8, 2016: 8, 2017: 8, 2024: 8, 2025: 8
     },
+    # 2020 Altima Platimum 1.8 Turbo
```
Comment (adeebshihadeh) [inline selfdrive/car/nissan/values.py:55]: merge with the existing fingerprint

**Comment by pd0wm** [comment]
Do you have a route from this car to verify the fingerprint?

**Comment by adeebshihadeh** [comment]
Any update on this?

**Comment by avolmensky** [comment]
Waiting on the route myself

**Comment by avolmensky** [comment]
Route: 08eaefeb586680da|2021-06-09--22-36-17

**Pull request merged**